src/Pages/Shared/ConfigMapSecret/ConfigMapSecret.reducer.ts

```diff
@@ -58,13 +58,27 @@ export const processCurrentData = (
     return [{ k: '', v: '', keyError: '', valueError: '' }]
 }
 
+const getExternalSubPathValues = (configMapSecretData: CMSecretConfigData) => {
+    const externalSubPathValues = {
+        value: configMapSecretData?.data ? Object.keys(configMapSecretData.data).join(',') : '',
+        error: '',
+    }
+
+    if (!externalSubPathValues.value && configMapSecretData?.esoSubPath) {
+        externalSubPathValues.value = configMapSecretData.esoSubPath ? configMapSecretData.esoSubPath.join(', ') : ''
```

Comment:
We should write this as
```suggestion
        externalSubPathValues.value = configMapSecretData.esoSubPath.join(', ') 
```